### ISSUE TITLE
the problem of the loss function converage

### ISSUE THREAD

**Issue by 9796l** [tjqansthd/LapDepth-release#22]
excuse me, i use other RGBD outdoor dataset,besides,the resolution is the same as NYU,i use the same training method as NYU in this code.But the loss function is not converage.becuse 30 epochs do not satisfy the converage,so i train the code using 60 epochs.but the loss function is not converage.the result is as below.train rmse is left.train_loss is right.
<img width="410" alt="1677409104316" src="https://user-images.githubusercontent.com/65001823/221406485-482a0b02-0802-4ee3-81a3-2753f68778bf.png">
<img width="439" alt="1677409295217" src="https://user-images.githubusercontent.com/65001823/221406517-a191133a-a2ca-4bf1-ac9b-eb85fd42bf74.png">
